tests/std/tests/P0896R4_views_transform_iterator/test.cpp

```diff
@@ -0,0 +1,339 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <ranges>
+#include <type_traits>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+#if _ITERATOR_DEBUG_LEVEL == 0

```

Comment:
Could you please move this back to `P0896R4_views_transform`? We're trying to avoid introducing gratuitous changes while implementing these super-late C++20 changes just in case we need to backport them to 16.11.

(To be clear: I don't mind this kind of "cleanup" change in general, I'd just like it to happen after we finish implementing the potential backport work.)